<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>716</width>
    <height>583</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Real Red Player</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: red;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="videocontainer" native="true">
    <property name="geometry">
     <rect>
      <x>39</x>
      <y>29</y>
      <width>561</width>
      <height>371</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="openButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>500</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Open</string>
    </property>
   </widget>
   <widget class="QPushButton" name="playButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>490</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Play</string>
    </property>
   </widget>
   <widget class="QSlider" name="durationslider">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>500</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="durationText">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>500</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
